internal/constraints/constraints.go

```diff
@@ -400,21 +390,7 @@ func osIsConstrained(constraintOSes string) bool {
 // Returns whether or not the current platform is constrained by the given
 // named constraints, which are defined in the given project configuration.
 func platformIsConstrained(constraintNames string) bool {
-	project := projectfile.Get()
-	names := strings.Split(constraintNames, ",")
-	constrained := true
-	for _, name := range names {
-		for _, platform := range project.Platforms {
-			if platform.Name == strings.TrimLeft(name, "-") && PlatformMatches(platform) {
-				if strings.HasPrefix(name, "-") {
-					return true
-				}
-				constrained = false // can't return here because an exclude might still occur
-			}
-		}
-	}
-
-	return constrained
+	return false // project.Platforms no longer exists
```

Comment:
Is this function still needed? Can we remove the platform constraint as well?

Reply:
This is why I filed https://activestatef.atlassian.net/browse/DX-1455. I'd rather remove it there since platform constraints still technically exist.